docs/Images.md

```diff
@@ -1,54 +1,79 @@
-## Static Resources
+---
+id: images
+title: Images
+layout: docs
+category: Guides
+permalink: docs/images.html
+next: gesture-responder-system
+---
 
-Over the course of a project, it is not uncommon to add and remove many images and accidentally end up shipping images you are no longer using in the app. In order to fight this, we need to find a way to know statically which images are being used in the app. To do that, we introduced a marker on require. The only allowed way to refer to an image in the bundle is to literally write `require('image!name-of-the-asset')` in the source.
+## Static Image Resources
+
+As of 0.14 release, React Native provides a unified way of managing images in your iOS and Android apps. To add a static image to your app, place it somewhere in your source code tree and reference it like this:
 
 ```javascript
-// GOOD
-<Image source={require('image!my-icon')} />
+<Image source={require('./my-icon.png')} />
+```
 
-// BAD
-var icon = this.props.active ? 'my-icon-active' : 'my-icon-inactive';
-<Image source={require('image!' + icon)} />
+Image name is resolved the same way JS modules are resolved. In the example above the packager will look for `my-icon.png` in the same folder as the component that requires it. Also if you have `my-icon.ios.png` and `my-icon.android.png`, the packager will pick the file depending on the platform you are running on.
 
-// GOOD
-var icon = this.props.active ? require('image!my-icon-active') : require('image!my-icon-inactive');
-<Image source={icon} />
+You can also use `@2x`, `@3x`, etc. suffix in the file name to provide images for different screen densities. For example, if you have the following file structure:
+
+```
+.
+├── button.js
+└── img
+    ├── check@2x.png
+    └── check@3x.png
 ```
 
-When your entire codebase respects this convention, you're able to do interesting things like automatically packaging the assets that are being used in your app. Note that in the current form, nothing is enforced, but it will be in the future.
+And `button.js` code contains
 
-### Adding Static Resources to your iOS App using Images.xcassets
+```javascript
+<Image source={require('./img/check.png')} />
+```
 
-![](/react-native/img/StaticImageAssets.png)
+Packager will bundle and server image corresponding to device's screen density, e.g. on iPhone 5s `check@2x.png` will be used, on Nexus 5 – `check@3x.png`. If there is no image matching screen density, closest best option will be selected.
```

Comment:
> bundle and server image

bundle and serve the image

> If there is no image matching screen density, closest

If there is no image matching the screen density, the closest